stats/stats-server/src/server.rs

```diff
@@ -89,7 +83,10 @@ pub async fn stats(mut settings: Settings) -> Result<(), anyhow::Error> {
         .await
         .context("stats DB")?,
     );
+    Ok(db)
+}
 
+async fn init_blockscout_db(settings: &Settings) -> anyhow::Result<Arc<DatabaseConnection>> {
```

Comment:
you init stats db that is correct, but you should only **connect** to blockscout db, to show that blockscout database is read only